test/extensions/config/validators/minimum_clusters/config_test.cc

```diff
@@ -0,0 +1,52 @@
+#include "envoy/extensions/config/validators/minimum_clusters/v3/minimum_clusters.pb.h"
```

Comment:
Is there some test to check if we have multiple validators that later ones will be called?

Reply:
I've got the following unit test that tests 2 generic validators: https://github.com/envoyproxy/envoy/blob/f2260c6c4eb688cacf2d6398c71b22698f4534fd/test/common/config/custom_config_validators_impl_test.cc#L141-L160

Let me know if you want me to add an integration test that actually runs 2 Minimum-Clusters with different thresholds.